frontend/src/ClassList.svelte

```diff
@@ -88,9 +88,11 @@
           <span class="iconify" data-icon="mdi:calendar-import"></span>
         </a>
 
+        {#if $user.is_staff}
```

Comment:
What does `$` do here? :)

Reply:
I have no idea, but without it, it doesn't work at all :)

Reply:
Well, IMO, it forces `user` to load actually.

Reply:
Ok :)